### PR TITLE
Added Lao support

### LINKED ISSUE #132: Lao Language support
Hello, thanks for this awesome lib.  
Would you be interested in adding support for Lao language? (p.s.: "milliseconds" seems to be non-existent in Laotian)

```javascript
var languages = {
  lo: {
    y: 'ປີ',
    mo: 'ເດືອນ',
    w: 'ອາທິດ',
    d: 'ມື້',
    h: 'ຊົ່ວໂມງ',
    m: 'ນາທີ',
    s: 'ວິນາທີ',
    ms: function (c) { return 'millisecond' + (c === 1 ? '' : 's') },
    decimal: ','
  }
}
```
Translation credits to: `Malikoun`

Right now I'm just using a custom humanizer with it defined, and works great. I can provide a PR next week or so if you want.

### PR DESCRIPTION
**Pull request by sam-lex**
Closes #132 

Hey there, I was able to get to this sooner than I thought...
Tests are based on English language, because there seems to have no "milliseconds" in Lao, `ms` will return the English version `millisecond(s)`.  
Lao doesn't have plurals, it's pretty much like Chinese.  

In Readme, I credited to `Malikoun` because she was the one that translated.

Thank you very much 😄

### CODE REVIEW

File: humanize-duration.js [hunk 1]
```diff
@@ -200,6 +200,17 @@
...
     },
+    lo: {
+      y: 'ປີ',
+      mo: 'ເດືອນ',
+      w: 'ອາທິດ',
+      d: 'ມື້',
+      h: 'ຊົ່ວໂມງ',
+      m: 'ນາທີ',
+      s: 'ວິນາທີ',
+      ms: function (c) { return 'millisecond' + (c === 1 ? '' : 's') },
```
[the remark this answers is not in the document]
Comment (sam-lex) [reply humanize-duration.js]: Sure, here's some info that I gathered, none of the dictionaries that we looked through had translations for `millisecond`.

Some dictionaries will translate `milli` to `ມິນລິ` *(which when pronounced, it sounds like `milli`. Just like how `cake` in Japanese is: `ケーキ Kēki`)*

For example, Google translates:
- millimeter: ມິນລິເມດ => `milli` + `med` (meter)
- milliliter: ມິນລິລິດ => `milli` + `lid` (liter)

By logic, we could combine `milli` + `seconds` = `ມິນລິວິນາທີ`. Google translate it **correctly from Lao to English**, but not from English to Lao, this is **probably correct**, but I couldn't get any confirmation on this...  
I asked a few Lao friends, and none of them could understand what it meant *(they're unrelated to IT or anything that needs millisecond precision, it was actually the first time they heard about milliseconds)*.  
Most of the time they will use the English version for words that they don't natively have.


A few dictionaries (older dictionaries) translates `millimeters`, `milliliters` into:  
- 0,001 **ຂອງ** X (`ຂອງ` means `of`, so: 0,001 of meters/liters)

So options would be:
1. `32 milliseconds` (English version)
2. `32 ມິນລິວິນາທີ`: 32 milliseconds (logically correct, but unconfirmed)
3. `0,032 ວິນາທີ`: 0,032 seconds
4. `0,032 ຂອງວິນາທີ`: 0,032 **of** seconds

let me know what you think  
Thanks 😄

[the remark this answers is not in the document]
Comment (sam-lex) [reply humanize-duration.js]: I'll change it to `ມິນລິວິນາທີ`. Just occurred to me to search for `milliseconds` in Thai.  
Lao is very similar to Thai, just like Spanish and Portuguese.  
`มิลลิวินาที - Milli wināthī` (Thai)
`ມິນລິວິນາທີ - minli vinathi` (Lao)

[the remark this answers is not in the document]
Comment (EvanHahn) [reply humanize-duration.js]: This makes sense to me (though I know nothing about Lao)! I'll get to accepting and releasing this soon.

**Comment by EvanHahn** [comment]
Thanks! This is now in `humanize-duration@3.15.0`.

**Pull request merged**